src/bb-functions-publish.ts

```diff
@@ -189,7 +198,7 @@ const groomMetaData = async (): Promise<MetaData> => {
 const publishFunctions = async (
   metaData: MetaData,
 ): Promise<string | object | null> => {
-  const ide = new IDE(identifier);
+  const ide = new IDE(identifier, targetHost);
```

Comment:
With this change, I don't think we need to pass the `identifier` any more? Since it was only used in the constructor to get the host.

Reply:
You are correct.
I brainfarted and kept it around to determine the build directory. 
But that's in another function.

Cleaned it up!